### DISCUSSION TITLE
Realearn: All keyboard notes to the same pan target

### DISCUSSION THREAD

**Discussion by PotoPolo01** [helgoboss/helgobox#1306] [General]
Hi, thanks first for the great app and work:)

My case and question:
I'm working on generative music and need to randomly affect pan, on any MIDI note played. So I'd like to use Realearn to connect all the notes of a keyboard, to a rand control.
Is there a way that I missed to connect all the notes at once ?
A way to group sources, by mean of range or something ?
Do I have to map all them 8 octaves one by one ?

Tell me the truth but don't be too brutal please:)

**Comment by helgoboss** [comment]
For this, you should generate ReaLearn presets via Luau (via a for loop and randomization functions). There's some chapter about ReaLearn's Luau support in the docs.

**Reply by PotoPolo01** [reply]
Aaargh !
Thanks a lot HelgoGod:)
